src/sec_related_rates.mbx

```diff
@@ -269,232 +269,303 @@
-
             <ol>
-              <li><p>The officer is traveling due north at 50mph and is 1/2 mile from the intersection, while the other car is 1 mile from the intersection traveling west and the radar reading is <m>-80</m>mph?</p></li>
-
-              <li><p>The officer is traveling due north at 50mph and is 1 mile from the intersection, while the other car is 1/2 mile from the intersection traveling west and the radar reading is <m>-80</m>mph?</p></li>
+              <li>
+                <p>The officer is traveling due north at 50mph and is 1/2 mile from the intersection, while the other car is 1 mile from the intersection traveling west and the radar reading is <m>-80</m>mph?</p>
+                <p><var name="$a" width="10"/></p>
+              </li>
+              <li>
+                <p>The officer is traveling due north at 50mph and is 1 mile from the intersection, while the other car is 1/2 mile from the intersection traveling west and the radar reading is <m>-80</m>mph?</p>
+                <p><var name="$b" width="10"/></p>
+              </li>
             </ol>
           </statement>
-          <solution>
-            <ol>
-              <li><p><m>64.44</m> mph</p></li>
-
-              <li><p><m>78.89</m> mph</p></li>
-            </ol>
-          </solution>
+<!--FUNGUS: There's no reason I need to keep the <solution> text, is there?-->
         </webwork>
       </exercise>
+<!--Exercise 7: Anti-aircraft gun-->
+<!--FUNGUS: Why does this exercise have xml:id? The others do not.-->
       <exercise xml:id="exer_04_02_ex_07">
         <webwork seed="1">
           <setup>
-            <var name="">
-              <static></static>
+<!--FUNGUS: Should I solve these to get the requisite 4 significant figures?-->
+            <var name="$a">
+              <static>0.0573</static>
+            </var>
+            <var name="$b">
+              <static>0.0725</static>
+            </var>
+            <var name="$c">
+              <static>0.0733</static>
             </var>
             <pg-code>
+              $a = 0.0573;
+              $b = 0.0725;
+              $c = 0.0733;
             </pg-code>
           </setup>
           <statement>
+<!--FUNGUS: With the picture, this question seems okay, but without it, the aircraft's minimum distance is  10,000 feet away. The ground distance should be referenced, for maximum clarity.-->
+<!--Except, there's no code here to embed the picture like there is for the question with the ladder. But there is a picture in the printed book I have.-->
+<!--FUNGUS: This problem makes no sense. You have to lead a target to actually hit it. Are we assuming the gun rotates to a spot in front of the aircraft that accounts for travel time of aircraft, projectile, and detonation? Real-world problems should be actually real. Make it a camera tracking a plane or something instead.-->
```

Comment:
I don't, but lets' not get bogged down with editing the author's work to that degree. For now, do your best to make it work how you think he intended.